src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -168,6 +169,36 @@ const parseProgramStatusGQLResp = (data: SideMenuProgramStatusQuery | undefined)
 	};
 };
 
+const renderDataSubmissionLinks = (
+	programName: string,
+	statusIcon: EmotionJSX.Element | null,
```

Comment:
can `React.ReactNode` be used here? I think we should avoid depending on EmotionJSX if possible. Their typings aren't always inline with Reacts typings.

Reply:
You're right, Emotion coupling is something we'd want to keep to a minimum; and considering the type you suggest would allow strings, etc. I think more than a node, an icon needs to be an element.

Reply:
Yes thankfully we can. I used Emotion as it was the suggested type in VSCode

<img width="523" alt="Screenshot 2023-11-28 at 10 27 10 AM" src="https://github.com/icgc-argo/rdpc-ui/assets/15621876/2d4fdf72-9c06-4a72-b12c-f5d90dfca79f">